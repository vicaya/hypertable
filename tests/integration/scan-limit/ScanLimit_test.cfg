#
# hypertable.cfg
#

# Global properties
Hypertable.Request.Timeout=180000

# HDFS Broker
HdfsBroker.Port=38030
HdfsBroker.fs.default.name=hdfs://localhost:9000
HdfsBroker.Workers=20

# Local Broker
DfsBroker.Local.Port=38030
DfsBroker.Local.Root=fs/local

# DFS Broker - for clients
DfsBroker.Host=localhost
DfsBroker.Port=38030

# Hyperspace
Hyperspace.Replica.Host=localhost
Hyperspace.Replica.Port=38040
Hyperspace.Replica.Dir=hyperspace
Hyperspace.Replica.Workers=20

# Hypertable.Master
Hypertable.Master.Host=localhost
Hypertable.Master.Port=38050
Hypertable.Master.Workers=20


# Hypertable.RangeServer
Hypertable.RangeServer.Port=38060
Hypertable.RangeServer.AccessGroup.MaxMemory=10
Hypertable.RangeServer.Maintenance.Interval=1

Hyperspace.KeepAlive.Interval=30000
Hyperspace.Lease.Interval=1000000
Hyperspace.GracePeriod=200000

# ThriftBroker
ThriftBroker.Port=38080
