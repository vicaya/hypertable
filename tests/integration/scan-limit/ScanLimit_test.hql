use '/';
drop table if exists ScanLimitTable;
create table ScanLimitTable(
  Field,
  ACCESS GROUP default bloomfilter='rows --false-positive 0.01'
) COMPRESSOR="none";
insert into ScanLimitTable VALUES ("Ensure Hypertable.RangeServer.AccessGroup.MaxMemory",
    'Field', " is less than the size of this row");
insert into ScanLimitTable VALUES ("In which case queries will hit the CellStore", 'Field',
    " and NOT the CellCache, since it is too small");
pause 5;
select * from ScanLimitTable limit=1;
select * from ScanLimitTable limit=2;
