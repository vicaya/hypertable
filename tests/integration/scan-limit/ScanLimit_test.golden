
Welcome to the hypertable command interpreter.
For information about Hypertable, visit http://www.hypertable.org/

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

use '/';
drop table if exists ScanLimitTable;
create table ScanLimitTable(
Field,
ACCESS GROUP default bloomfilter='rows --false-positive 0.01'
) COMPRESSOR="none";
insert into ScanLimitTable VALUES ("Ensure Hypertable.RangeServer.AccessGroup.MaxMemory",
'Field', " is less than the size of this row");
insert into ScanLimitTable VALUES ("In which case queries will hit the CellStore", 'Field',
" and NOT the CellCache, since it is too small");
pause 5;
select * from ScanLimitTable limit=1;
Ensure Hypertable.RangeServer.AccessGroup.MaxMemory	Field	 is less than the size of this row
select * from ScanLimitTable limit=2;
Ensure Hypertable.RangeServer.AccessGroup.MaxMemory	Field	 is less than the size of this row
In which case queries will hit the CellStore	Field	 and NOT the CellCache, since it is too small
